assets/src/apps/delivery/components/ActivityRenderer.tsx

```diff
@@ -420,7 +420,33 @@ const ActivityRenderer: React.FC<ActivityRendererProps> = ({
     }
     notifyStateMutation();
   }, [mutationTriggered]);
+  const debounceStateChanges = async (changes: any) => {
```

Comment:
not sure why it is called `debounceStateChanges` as there is no debounce, maybe just `handleStateChangeEvents`

Reply:
Done